src/scene/mesh.js

```diff
@@ -95,6 +95,7 @@ Object.assign(pc, function () {
      * @property {number} drawOrder Use this value to affect rendering order of mesh instances.
      * Only used when mesh instances are added to a {@link pc.Layer} with {@link pc.Layer#opaqueSortMode} or {@link pc.Layer#transparentSortMode} (depending on the material) set to {@link pc.SORTMODE_MANUAL}.
      * @property {boolean} visibleThisFrame Read this value in {@link pc.Layer#onPostCull} to determine if the object is actually going to be rendered.
+     * @property {pc.callbacks.CalculateSortDistance} calculateSortDistance Set this value to override the default calculation for the "sort distance" for this mesh instance, used to determine its place in the render order.

```

Comment:
Would be good to document what arguments are passed to the method.

Reply:
Don't you see those when you click through `pc.callbacks.CalculateSortDistance` in the API reference?

Reply:
@willeastcott, yes, I too think you'd see the arguments when you click through to `pc.callbacks.CalculateSortDistance` in the API reference.

@Maksims, do you prefer to list the same information twice, i.e. once on the API reference page for `pc.callbacks.CalculateSortDistance` and once more on the page for `pc.MeshInstance.calculateSortDistance`? I'd prefer a [single source of truth](https://en.wikipedia.org/wiki/Single_source_of_truth).
However, I noticed that the arguments for at least one other callback `pc.callbacks.CalculateMatrix` is listed not just twice but at least _thrice_ ([1](https://developer.playcanvas.com/en/api/pc.callbacks.html#CalculateMatrix), [2](https://developer.playcanvas.com/en/api/pc.CameraComponent.html#calculateProjection), [3](https://developer.playcanvas.com/en/api/pc.CameraComponent.html#calculateTransform)).

To follow the pattern I'll update the PR to add information about the new callback in multiple places in the same manner.

Reply:
No, I'd prefer you don't duplicate and follow the 'one source of truth' path please. If you see examples of the docs breaking that style, we should fix them.

Reply:
Ok, I removed the duplicate documentation! 👍